assets/scss/main.scss

```diff
@@ -14,3 +14,9 @@
 //
 @import "accordion";
 @import "popover";
+
+//
+// SWARM-ANIMATION
+//
+@import "~swarm-animation/src/constants";
+@import "~swarm-animation/src/fade";
```

Comment:
actually, we may want to import these in the component partials that depend on them